packages/generator-langium/templates/web/.package.json

```diff
@@ -1,23 +1,19 @@
 {
     "scripts": {
-        "build:worker": "esbuild --minify ./out/language/main-browser.js --bundle --format=iife --outfile=./public/<%= language-id %>-server-worker.js",
-        "prepare:public": "shx mkdir -p ./public && shx cp -fr ./src/static/* ./public/",
-        "copy:monaco-editor-wrapper": "shx cp -fr ./node_modules/monaco-editor-wrapper/bundle ./public/monaco-editor-wrapper",
-        "copy:monaco-workers": "shx cp -fr ./node_modules/monaco-editor-workers/dist/ ./public/monaco-editor-workers",
-        "copy:monarch-syntax": "shx cp -fr ./out/syntaxes ./public/syntaxes",
-        "build:web": "npm run build && npm run build:monarch && npm run prepare:public && npm run build:worker && npm run copy:monaco-editor-wrapper && npm run copy:monaco-workers && npm run copy:monarch-syntax",
+        "prepare:static": "node ./scripts/prepare.js",
+        "build:web": "npm run build && npm run prepare:static && npm run build:monarch && npm run build:worker && npm run build:bundles",
         "build:monarch": "tsc -b tsconfig.monarch.json",
-        "serve": "node ./out/web/app.js"
+        "build:bundles": "vite --config ./vite.bundle.classic.config.ts build && vite --config ./vite.bundle.vscodeApi.config.ts build",
+        "build:worker": "esbuild --minify ./out/main-browser.js --bundle --format=iife --outfile=./static/worker/<%= language-id %>-server-worker.js",
```

Comment:
This didn't work when I tested things out locally. The main browser is still in `./out/language/main-browser.js`, but this was the only hiccup I found while testing out the generator.

Reply:
This is weird. Both Windows and Linux produce the dir structure as seen in the screenshot and therefore I don't see that error. I changed the command, because I had the opposite problem 🤷‍♂️ Did you use `npm run run`? 

![image](https://github.com/eclipse-langium/langium/assets/21079138/9459fa09-762e-4d6f-9db9-5f12ab8a9bb0)

Reply:
Yep, was working through `npm run run`, and post-clean as well. Possible difference due to platforms?

Reply:
Fixed in next commit.